src/views/ItemView.test.js

```diff
@@ -122,5 +138,40 @@ describe('ItemView', () => {
       expect(document.querySelector('#item-list-bound-with-titles a.instanceHrid'))
         .toHaveAttribute('href', '/inventory/view/' + id);
     });
+
```

Comment:
I would love feedback on whether all of these tests are necessary, or on the other hand insufficient.  They seem insufficient since to test fully we'd have to mock up maybe a dozen different item records to really test the presence (or absence) of the Inactive flag.  But on the other hand they seem unnecessary in that there are very few tests for any other read-only data on the item record, unless that itself is a problem we're trying to fix.

On the other hand if this is really what's intended I'd like to know that as well!